apps/coscrad-frontend/src/components/AllResources/ResourceInfos.presenter.tsx

```diff
@@ -12,10 +13,11 @@ export const ResourceInfosPresenter: FunctionalComponent<
 > = ({ data: resourceInfos }: HasData<IAggregateInfo<ResourceType>[]>) => (
     <div>
         <h2>Available Resources</h2>
-        <div>
-            {resourceInfos.map((info) => (
-                <ResourceInfoPresenter {...info} key={info.type}></ResourceInfoPresenter>
-            ))}
-        </div>
+        <Divider />
+        <br />
+        <br />
+        {resourceInfos.map((info) => (
```

Comment:
Wrap this in a MUI stack with `spacing={1}`

Reply:
done